tests/e2e/csi_cns_telemetry.go

```diff
@@ -167,49 +174,67 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-file-vanilla] CNS-CSI Cluster
+
+		// For Old PVCs to reflect the latest Cluster-Distribution Value, wait for
+		// full sync.
+		framework.Logf("Sleeping full-sync interval for all the volumes Metadata " +
+			"to reflect the cluster-distribution value to = Empty")
 		time.Sleep(time.Duration(fullSyncWaitTime) * time.Second)
 
-		//Add additional safe wait time for cluster-distribution to reflect on metadata
-		framework.Logf("Sleeping for additional safe one min for volumes Metadata to reflect the cluster-distribution value to = Empty")
+		// Add additional safe wait time for cluster-distribution to reflect on
+		// metadata.
+		framework.Logf("Sleeping for additional safe one min for volumes Metadata " +
+			"to reflect the cluster-distribution value to = Empty")
 		time.Sleep(time.Duration(pollTimeoutShort))
 
 		ginkgo.By(fmt.Sprintf("Invoking QueryCNSVolumeWithResult for PVC1 with VolumeID: %s", volHandle1))
 		queryResult3, err := e2eVSphere.queryCNSVolumeWithResult(volHandle1)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(len(queryResult3.Volumes) > 0).To(gomega.BeTrue())
 
-		framework.Logf("Cluster-distribution value on CNS is %s", queryResult3.Volumes[0].Metadata.ContainerClusterArray[0].ClusterDistribution)
-		gomega.Expect(queryResult3.Volumes[0].Metadata.ContainerClusterArray[0].ClusterDistribution).Should(gomega.Equal(""), "Wrong/empty cluster-distribution name present on CNS")
+		framework.Logf("Cluster-distribution value on CNS is %s",
+			queryResult3.Volumes[0].Metadata.ContainerClusterArray[0].ClusterDistribution)
+		gomega.Expect(queryResult3.Volumes[0].Metadata.ContainerClusterArray[0].ClusterDistribution).Should(
+			gomega.Equal(""), "Wrong/empty cluster-distribution name present on CNS")
 
 		ginkgo.By("Setting the cluster-distribution value with escape character and special characters")
 		setClusterDistribution(ctx, client, vanillaClusterDistributionWithSpecialChar)
 
-		// For Old PVCs to reflect the latest Cluster-Distribution Value, wait for full sync
-		framework.Logf("Sleeping full-sync interval for all the volumes Metadata to reflect the cluster-distribution value to = %s", vanillaClusterDistributionWithSpecialChar)
+		// For Old PVCs to reflect the latest Cluster-Distribution Value, wait for
+		// full sync.
+		framework.Logf("Sleeping full-sync interval for all the volumes Metadata to reflect "+
+			"the cluster-distribution value to = %s", vanillaClusterDistributionWithSpecialChar)
 		time.Sleep(time.Duration(fullSyncWaitTime) * time.Second)
 
-		//Add additional safe wait time for cluster-distribution to reflect on metadata
-		framework.Logf("Sleeping for additional safe one min for volumes Metadata to reflect the cluster-distribution value to = %s", vanillaClusterDistributionWithSpecialChar)
+		// Add additional safe wait time for cluster-distribution to reflect on
+		// metadata.
+		framework.Logf("Sleeping for additional safe one min for volumes Metadata to reflect "+
```

Comment:
Add a space in front of "+"

Reply:
I am not sure how "go fmt" works. Sometimes, it removes the white space before "+" automatically.
golang is weird.